src/main/java/com/feedzai/commons/sql/abstraction/engine/AbstractDatabaseEngine.java

```diff
@@ -633,13 +643,32 @@ public synchronized void dropEntity(String entity) throws DatabaseEngineExceptio
      * @param entity The entity.
      * @throws DatabaseEngineException If something goes wrong while dropping the structures.
      */
+    @Override
     public synchronized void dropEntity(final DbEntity entity) throws DatabaseEngineException {
         dropSequences(entity);
         dropTable(entity);
         entities.remove(entity.getName());
         logger.trace("Entity {} dropped", entity.getName());
     }
 
+    /**
+     * Drops all entities associated with this engine.
+     *
+     * @since 2.1.13
+     */
+    private void dropAllEntities() {
+
+        for (MappedEntity mappedEntity : ImmutableList.copyOf(entities.values())) {
```

Comment:
`mappedEntity` can be final, and you have a few blank lines in this method that don't really improve legibility.